components/roadmap-grid/RoadmapDetailedView.tsx

```diff
@@ -33,13 +33,14 @@ export function RoadmapDetailed({
   const router = useRouter();
 
   const issuesGroupedState = useHookstate<DetailedViewGroup[]>([]);
-  // const [dayjsDates, setDayjsDates] = useState<Dayjs[]>([]);
 
+  const setIssuesGroupedState = issuesGroupedState.set
   useEffect(() => {
     if (viewMode) {
-      issuesGroupedState.set(convertIssueDataStateToDetailedViewGroupOld(issueDataState, viewMode, router.query))
+      setIssuesGroupedState(convertIssueDataStateToDetailedViewGroupOld(issueDataState, viewMode, router.query))
     }
-  }, [viewMode, issueDataState.value]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [viewMode, router.query, issueDataState.children]);
```

Comment:
this should eventually be moved to useMemo, but it's late and I wanted to close up shop